app/cloudinit/user_data/gcp_linux

```diff
@@ -0,0 +1,73 @@
+#cloud-config
```

Comment:
this file should not be added. ubuntu_linux is the one we use. gcp_linux we already removed from code.